lib/template/code_example_templates/js_example_template.html.erb

```diff
@@ -0,0 +1,7 @@
+<script><%= rendered_example %></script>
+<div class="codeBlock jsExample">
+  <div class="highlight">
+    <pre><%= code_example %></pre>
+  </div>
+</div>
+
```

Comment:
I like this, it seems much more reasonable for less technical users to contribute templates. If I understand correctly, they would only need to make a template file and a factory?